games-action/maelstrom/files/maelstrom-3.0.6-fix_return_type.patch

```diff
@@ -0,0 +1,17 @@
+Adds void to main()
+
+Quick fix to the compile issue on main. There is no return statement, so return type is set to void. 
+
+Signed-off-by: Stephane Bakhos <nuitari@nuitari.net>
+https://bugs.gentoo.org/875431
+--- a/Maelstrom-netd.c
++++ b/Maelstrom-netd.c
+@@ -184,7 +184,7 @@
+        exit(sig);
+ }
+ 
+-main(int argc, char *argv[])
```

Comment:
FWIW, this isn't right. main _has_ to return int, and it has an implicit int return (unlike any other function) even if not done explicitly. I'll fix that on merging.